src/components/profile/EditProfile.js

```diff
@@ -235,7 +235,7 @@ const getStylesFromProps = ({ theme }) => {
       borderRadius: halfAvatarSize,
     },
     userAvatar: {
-      borderWidth: 3,
+      borderWidth: -1,
```

Comment:
there's supposed to be a border. what is -1? you need to check that it matches today's look and feel on web.

Reply:
@MathiasMoralesDixtra

Reply:
On the web before changes, it looks bad also but you have the reason, I did not take into account the border. I'll check it.